lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -51,7 +51,13 @@ def self.deliver_queue(args, options = {})
   end
 
   def self.deliver(*args)
-    @deliver_mutex.synchronize { deliver_block(*args) }
+    # This was added because in RAILS 5 PUMA sends multiple requests
+    # to the same processa and our DRb server implementation for Automate
+    # Methods is not thread safe. The permit_concurrent_loads  allows the
+    # DRb requests which run in different threads to load symbols.
```

Comment:
@mkanoor Do you mean to "autoload models"? "load symbols" also doesn't make sense to me.